vhdbuilder/packer/components.json

```diff
@@ -162,25 +164,31 @@
       "downloadURL": "mcr.microsoft.com/oss/calico/typha:*",
       "versions": [
         "v3.17.2",
+        "v3.18.1",
+        "v3.19.0",
         "v3.8.9"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/calico/pod2daemon-flexvol:*",
       "versions": [
-        "v3.8.9"
+        "v3.8.9.1"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/calico/kube-controllers:*",
       "versions": [
-        "v3.17.2"
+        "v3.17.2",
+        "v3.18.1",
+        "v3.19.0"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/tigera/operator:*",
       "versions": [
-        "v1.13.5"
+        "v1.13.5",
+        "v1.15.1",
```

Comment:
Adding in operator v1.15.1 (and corresponding calico v3.18.1) images as requested by @andyliuliming.